Azure/sonic-buildimage
# Add support of HwSKU Mellanox-SN2700-C28D8

Signed-off-by: Andriy Moroz <c_andriym@mellanox.com>

**- What I did**

**- How I did it**

**- How to verify it**
1.	Installed some latest 201803 image we had
2.	Built and installed on switch sonic-device-data_1.0-1_all.deb with the new HwSKU configs on running switch (dpkg -i …)
3.	Updated minigraph to match port_config.ini
4.	Stopped services and removed syncd container
(not sure this is really needed, just to make sure it will pick up a new sai prorile)
5.	Loaded updated minigraph
6.	Test command line `sudo sfputil show presence` and there is no exception.

Ports were created as expected with the speed according to the schema provided by Qi.

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
